releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,21 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.2.0 - MM DD, 2025
+
+### Enhancements
+
+* Added support for on-demand refresh, which enables you to trigger an on-demand refresh to get the latest data from App Configuration and Key Vault by modifying `metadata.annotations` section. This enhancement makes the refresh process more flexible and responsive to real-time needs. [#87](https://github.com/Azure/AppConfiguration-KubernetesProvider/pull/87)

```

Comment:
Added support for on-demand refresh of ConfigMap and Secret. While you can set up automatic data refresh, there are times when you might want to trigger an on-demand refresh to get the latest data from App Configuration and Key Vault. To do this, you can modify the `metadata.annotations` section. This enhancement makes the refresh process more flexible and better suited to your workflow.